yarn.lock

```diff
@@ -1470,11 +1470,16 @@
     "@types/marked" "*"
     chalk "^2.4.1"
 
-"@types/marked@*", "@types/marked@^1.2.1":
+"@types/marked@*":
   version "1.2.1"
   resolved "https://registry.yarnpkg.com/@types/marked/-/marked-1.2.1.tgz#9864ecc10d0edb72e7be4e94acb5fcc607c15eed"
   integrity sha512-d5adCgRHB+NAme23hkiTkvpfZUDqoNtL2Sr2nZBJqSj3zyHLxsfFWsGQ2sK2z9aX6L1xkJzon2c0jTPcsEjpaQ==
 
+"@types/marked@^1.2.2":
```

Comment:
I hate yarn sometimes (after applying run `yarn` locally to make sure nothing else changes)
```suggestion
"@types/marked@*", "@types/marked@^1.2.2":
```